controllers/actions.github.com/constants.go

```diff
@@ -16,3 +18,47 @@ const (
 	EnvVarHTTPSProxy = "https_proxy"
 	EnvVarNoProxy    = "no_proxy"
 )
+
+// Labels applied to resources
+const (
```

Comment:
Moved label constants from resource builder to constants where I think is more appropriate